requirements.txt

```diff
@@ -2,4 +2,8 @@ Django==1.11.2
 Pillow==5.0.0
 WhiteNoise==3.3.1
 psycopg2==2.7.3.2
-django-access-tokens==0.9.2
\ No newline at end of file
+django-access-tokens==0.9.2
```

Comment:
@lauradobson what was the version you reinstalled  to??

Reply:
I think it was still the same one. I had to do "sudo pip install access-tokens"... Not quite sure what happened with that